<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseRemindDateTime</class>
 <widget class="QWidget" name="chooseRemindDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="chooseRemindDateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>296</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="removeRemindTime">
      <property name="text">
       <string>移除提醒</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chooseRemindTimeSureButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提醒时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="chooseRemindTimeEdit">
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
